File: JavCrawl/JavCrawl/Models/DbEntity/Tags.cs
using System;
using System.Collections.Generic;

namespace JavCrawl.Models.DbEntity
{
    public partial class Tags
    {
        public int Id { get; set; }
        public DateTime? CreatedAt { get; set; }
        public DateTime? DeletedAt { get; set; }
        public string Seo { get; set; }
        public string Slug { get; set; }
        public bool? Status { get; set; }
        public string Title { get; set; }
        public string TitleAscii { get; set; }
        public DateTime? UpdatedAt { get; set; }
    }
}

File: javmile.com/public_html/app/Models/Tags.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\SoftDeletes;
use Cviebrock\EloquentSluggable\SluggableInterface;
use Cviebrock\EloquentSluggable\SluggableTrait;

class Tags extends Model
{
    use SoftDeletes;
    protected $dates = ['deleted_at'];
    protected $fillable = array('title', 'title_ascii', 'slug', 'seo', 'status', 'created_at', 'updated_at', 'deleted_at');

    public function films() {
        return $this->belongsToMany('\App\Models\Film', 'film_tags', 'tag_id', 'film_id')->with(['thumb', 'videos', 'quality', 'category']);
    }
    function articles() {
        return $this->belongsToMany('\App\Models\Articles', 'article_tags', 'tag_id', 'article_id')->with('cover')->orderBy('id', 'desc');
    }
}
